plz.el

```diff
@@ -778,7 +790,8 @@ argument passed to `plz--sentinel', which see."
              ((and status (guard (<= 200 status 299)))
               ;; Any 2xx response is considered successful.
               (ignore status) ; Byte-compiling in Emacs <28 complains without this.
-              (funcall (process-get process :plz-then)))
+              (when-let (then (process-get process :plz-then))
```

Comment:
This is because I set the THEN callback to nil above when we stream, but no THEN is given. Let's see how we decide what do do about sync/async and revist this then.